.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           name: Run Showcase
           command: |
             mkdir protobuf
-            curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v3.11.4/protoc-3.11.4-linux-x86_64.zip > protobuf/protoc.zip
+            curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0-rc1/protoc-3.12.0-rc-1-linux-x86_64.zip > protobuf/protoc.zip
```

Comment:
Please add TODOs here and in other places, which either depend on `rc` verisons of dependencies or result of "hacks" (like overriding protobuf from `1.3.3` to `1.4.0`) briefly explaining when and it should be replaced with proper GA versions.

Reply:
Done. Good idea, thanks.